### PR TITLE
Count VVA as available if it does not raise an error

### PR DESCRIPTION
**Pull request by lowellrex**
Fixes https://github.com/department-of-veterans-affairs/appeals-deployment/issues/3359.

### CODE REVIEW

File: app/services/vva_service.rb [hunk 1]
```diff
@@ -15,8 +15,8 @@ def initialize
   end
 
   def query_service
-    documents = @client.document_list.get_by_claim_number(Rails.application.secrets.vva_file_num)
-    @pass = true if documents.size > 0
+    @client.document_list.get_by_claim_number(Rails.application.secrets.vva_file_num)
+    @pass = true
```
Comment (lowellrex) [inline app/services/vva_service.rb:19]: All failures in the call to `get_by_claim_number` (invalid login credentials, file number too short) should raise a `VVA::SOAPError`. If the call to `get_by_claim_number` succeeds then `@pass` will be `true`, even if there are no documents.

Comment (lowellrex) [reply app/services/vva_service.rb:19]: Not catching errors follows [the pattern established in `VacolsService`](https://github.com/department-of-veterans-affairs/caseflow-monitor/blob/a7e45c719483e5810b85d4078bff2d74a5b4aaaa/app/services/vacols_service.rb#L112), so I think we should be properly recording `@pass = false` conditions even though we aren't catching the exception here (I think [we are catching it in `MonitorJob`](https://github.com/department-of-veterans-affairs/caseflow-monitor/blob/a7e45c719483e5810b85d4078bff2d74a5b4aaaa/app/jobs/monitor_job.rb#L104)).

Comment (aroltsch) [reply app/services/vva_service.rb:19]: Shall we put your first comment in the code?

**Review by aroltsch — APPROVED** [review]
LGTM

File: app/services/vva_service.rb (hunk 1 above)
Comment (lowellrex) [reply app/services/vva_service.rb:19]: Added code comment in another PR: https://github.com/department-of-veterans-affairs/caseflow-monitor/pull/108

**Pull request merged**